<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pqSpreadSheetViewDecorator</class>
 <widget class="QWidget" name="pqSpreadSheetViewDecorator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>32</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;b&gt;Showing  &lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="pqOutputPortComboBox" name="Source">
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&lt;b&gt;   Attribute:&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="Attribute">
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_Precision">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Precision:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spinBoxPrecision">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="maximum">
      <number>32</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="ToggleFixed">
     <property name="toolTip">
      <string>Switches between scientific and fixed-point representation</string>
     </property>
     <property name="statusTip">
      <string>Toggle fixed-point representation (always show #Precision digits)</string>
     </property>
     <property name="whatsThis">
      <string>Toggle fixed-point representation (always show #Precision digits)</string>
     </property>
     <property name="text">
      <string>FixedRep</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/pqWidgets/Icons/pqFixedRepr32.png</normaloff>:/pqWidgets/Icons/pqFixedRepr32.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="SelectionOnly">
     <property name="toolTip">
      <string>Show only selected elements.</string>
     </property>
     <property name="statusTip">
      <string>Show only selected elements.</string>
     </property>
     <property name="whatsThis">
      <string>Show only selected elements.</string>
     </property>
     <property name="text">
      <string>Selected</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/pqWidgets/Icons/pqSelect16.png</normaloff>:/pqWidgets/Icons/pqSelect16.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonIconOnly</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="ToggleColumnVisibility">
     <property name="toolTip">
      <string>Toggle column visibility</string>
     </property>
     <property name="statusTip">
      <string>Toggle column visibility</string>
     </property>
     <property name="whatsThis">
      <string>Toggle column visibility</string>
     </property>
     <property name="text">
      <string>Toggle column visibility</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/pqWidgets/Icons/pqRectilinearGrid16.png</normaloff>:/pqWidgets/Icons/pqRectilinearGrid16.png</iconset>
     </property>
     <property name="popupMode">
      <enum>QToolButton::InstantPopup</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="ToggleCellConnectivity">
     <property name="toolTip">
      <string>Toggle cell connectivity visibility</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/pqWidgets/Icons/pqProgrammableFilter24.png</normaloff>:/pqWidgets/Icons/pqProgrammableFilter24.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="ExportSpreadsheet"/>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>58</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <action name="actionExport">
   <property name="icon">
    <iconset>
     <normaloff>:/pqWidgets/Icons/pqSaveTable32.png</normaloff>:/pqWidgets/Icons/pqSaveTable32.png</iconset>
   </property>
   <property name="toolTip">
    <string>Export Spreadsheet</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqOutputPortComboBox</class>
   <extends>QComboBox</extends>
   <header>pqOutputPortComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../pqComponents.qrc"/>
 </resources>
 <connections/>
</ui>
